data/yam/agama/auto/template.jsonnet

```diff
@@ -0,0 +1,20 @@
+local base_lib = import 'lib/base.libsonnet';
+local scripts = import 'lib/scripts.libsonnet';
+local storage = import 'lib/storage.libsonnet';
+
+function(storage_schema='', product='SLES', pre_scripts='',
```

Comment:
let's not invent words here, the only schema that we have is the one in the Agama repo against which the profile is validated, this should just be `storage`, we should make all the libraries looks like so, so above `local storage_lib` same for any other library.